README.md

```diff
@@ -1161,8 +1160,34 @@
     this.firstName_ = 'Panda';
 
     // good
-    this._firstName = 'Panda';
+    this._firstName = 'Panda';    
     ```
+ 
+  - Use UPPERCASE for nested object namespacing, global variables, and constants. 
```

Comment:
The note about global variables is a bit confusing. This could also use an explanation of what we mean by nested object namespacing.